packages/patternfly-4/react-core/src/components/Accordion/AccordionToggle.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@patternfly/react-styles';
+import { AngleRightIcon } from '@patternfly/react-icons';
+import styles from '@patternfly/patternfly/components/Accordion/accordion.css';
+import { Button } from 'patternfly-react';
+
+const AccordionToggle = ({
+  className,
+  'aria-controls': ariaControls,
+  'aria-label': ariaLabel,
+  'aria-labelledby': ariaLabelledBy,
+  id,
+  isExpanded,
+  children,
+  ...props
+}) => (
+  <dt>
+    <h3>
+      <Button
+        id={id}
+        className={css(styles.accordionToggle, isExpanded && styles.modifiers.expanded, className)}
+        {...props}
+        variant="plain"
+        aria-controls={ariaControls !== '' && ariaControls}
+        aria-label={ariaLabel}
+        aria-labelledby={ariaLabel !== 'Details' ? null : `${id}`}
+        aria-expanded={isExpanded}
+      >
+        <span className={css(styles.accordionToggleText)}>{children}</span>
+        <span className={css(styles.accordionToggleIcon)}>
```

Comment:
This class from core is missing the transition that is seen in other similar components. https://github.com/patternfly/patternfly-next/issues/1532 created for this.